xml/System.Resources.Extensions/DeserializingResourceReader.xml

```diff
@@ -24,7 +24,7 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides APIs similar to <see cref="T:System.Resources.ResourceReader" /> that can read and deserialize resource data.</summary>
```

Comment:
We could say written by either ResourceWriter or PreserializedResourceWriter.

Reply:
I'm not 100% sure where to put your suggestion. Do you mean changing it to this?:

```suggestion
    <summary>Provides APIs that can be written by either <see cref="T:System.Resources.ResourceWriter" /> or <see cref="T:System.Resources.Extensions.PreserializedResourceWriter" />.</summary>
```

Reply:
How about 
```suggestion
    <summary>Provides APIs similar to <see cref="T:System.Resources.ResourceReader" /> that can read and deserialize resource data written by either <see cref="T:System.Resources.ResourceWriter" /> or <see cref="T:System.Resources.Extensions.PreserializedResourceWriter" />.</summary>
```